bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx91-en.html

```diff
@@ -0,0 +1,105 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}What’s new with Firefox{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}What’s new with Firefox{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_91') }}
+{% endblock %}
+
+{% block experiments %}
+  {% if switch('firefox-whatsnew90-experiment', ['en-US', 'en-CA']) %}
+    {{ js_bundle('firefox_whatsnew90_experiment') }}
+  {% endif %}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew91' %}
+{% set utm_campaign = campaign + '-en' %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-md">
+      <img class="wnp-main-image" src="{{ static('protocol/img/logos/pocket/logo-word-hor.svg') }}" width="220" alt="Pocket">
+      <h1 class="wnp-main-title">Discover the best articles from across the web</h1>
+      <p class="wnp-main-tagline">Pocket shows you the hidden gems — stories from trusted publications that dive deeper, surface new perspectives and make you think.</p>
+
+      <div class="wnp-main-cta">
+        {{ pocket_fxa_button(entrypoint=_entrypoint, button_text='Activate Pocket in Firefox', optional_parameters={'s': _entrypoint, 'utm_campaign': utm_campaign, 'utm_content': 'signup-fxa-primary'}, optional_attributes={'data-cta-text': 'Activate Pocket in Firefox', 'data-cta-type': 'button', 'data-cta-position': 'primary'}) }}
+        <p class="c-sub-cta"><a href="https://getpocket.com/signup/?utm_source={{ _entrypoint }}&utm_medium=referral&utm_campaign={{ utm_campaign }}" data-cta-text="More ways to sign up" data-cta-position="primary-sub" data-cta-type="link">More ways to sign up</a></p>
+      </div>
+    </div>
+  </section>
+
+  <section class="wnp-content-extra">
+    <div class="mzp-l-content mzp-t-content-lg">
+      <div class="mzp-l-columns mzp-t-columns-three">
+
+        <div class="mzp-c-picto">
+          <div class="mzp-c-picto-image">
+            <img src="{{ static('img/firefox/whatsnew/whatsnew91/person1.svg') }}" alt="">
```

Comment:
Went from the inline SVG to embedded images because the new illustrations were tweaked to work on both light and dark backgrounds.